flaml/automl/model.py

```diff
@@ -2028,6 +2028,104 @@ def fit(self, X_train, y_train, budget=None, free_mem_ratio=0, **kwargs):
         return train_time
 
 
+class HoltWinters(ARIMA):
+    """
+    The class for tuning Holt Winters model, aka 'Triple Exponential Smoothing'.
+    """
+
+    @classmethod
+    def search_space(cls, **params):
+        space = {
+            "damped_trend": {
+                "domain": tune.choice([True, False]),
+                "init_value": False,
+                "low_cost_init_value": False,
+            },
+            "trend": {
+                "domain": tune.choice(["add", "mul", None]),
+                "init_value": "add",
+                "low_cost_init_value": "add",
+            },
+            "seasonal": {
+                "domain": tune.choice(["add", "mul", None]),
+                "init_value": "add",
+                "low_cost_init_value": None,
+            },
+            "use_boxcox": {
+                "domain": tune.choice([False, True]),
+                "init_value": False,
+                "low_cost_init_value": True,
+            },
+            "seasonal_periods": {  # statsmodels casts this to None if "seasonal" is None
+                "domain": tune.choice(
+                    [7, 12, 4, 52, 6]
+                ),  # weekly, yearly, quarterly, weekly w yearly data
+                "init_value": 7,
+                "low_cost_init_value": 7,
+            },
+        }
+        return space
+
+    def fit(self, X_train, y_train, budget=None, free_mem_ratio=0, **kwargs):
+        import warnings
+
+        warnings.filterwarnings("ignore")
+        from statsmodels.tsa.holtwinters import (
+            ExponentialSmoothing as HWExponentialSmoothing,
+        )
+
+        current_time = time.time()
+        train_df = self._join(X_train, y_train)
+        train_df = self._preprocess(train_df)
+        regressors = list(train_df)
+        regressors.remove(TS_VALUE_COL)
+        if regressors:
+            # Not supported ATM. A few discussions here:
+            # https://robjhyndman.com/hyndsight/ets-regressors/
+            # https://stats.stackexchange.com/questions/220830/holt-winters-with-exogenous-regressors-in-r
+            logger.debug("Regressors are ignored for Holt-Winters ETS models.")

```

Comment:
What about
```suggestion
            logger.warning("Regressors are ignored for Holt-Winters ETS models.")
```

Reply:
I'm happy to accept the suggestion, but is there a place where we can document this kind of choice? I'd personally find it helpful. Just a humble proposal, you may have good reasons for not being willing to maintain such kind of documentation

Reply:
Thanks for this proposal. We should consider adding such documentation. Perhaps under https://microsoft.github.io/FLAML/docs/Contribute or a subpage.